pyproject.toml

```diff
@@ -3,7 +3,15 @@ name = "poetry"
 version = "1.2.0-beta.2.dev0"
 description = "Python dependency management and packaging made easy."
 authors = [
-    "Sébastien Eustace <sebastien@eustace.io>"
+    "Sébastien Eustace <sebastien@eustace.io>",
+    "Arun Babu Neelicattu <arun.neelicattu@gmail.com>",
```

Comment:
Wouldn't it fit better to leave sdispater as author and add rest of the team as `maintainers`?

Reply:
You know, I forgot that field existed in the Python metadata...